BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/SkontoViewModel.swift

```diff
@@ -52,6 +52,16 @@ class SkontoViewModel {
         )
     }
 
+    var savedAmountString: String {
+        let savedAmount = calculateSavedAmount()
+        guard let priceString = savedAmount.localizedStringWithCurrencyCode else { return "" }
+        return String.localizedStringWithFormat(
+            NSLocalizedStringPreferredGiniBankFormat("ginibank.skonto.total.saved",
```

Comment:
[suggestion] Should we use "savings" everywhere instead of saved? When I'm using saved is more of a past action to save a modification and not that I'm saving money like in our case.